tox.ini

```diff
@@ -1,27 +1,46 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,pypy3,docs
+    py27,
+    py33,
+    py34,
+    py35,
+    py36,
+    pypy,
+    pypy3,
+    docs,
+    coverage-report,
 
 [testenv]
+usedevelop = true
```

Comment:
@mgedmin You are right, using this configuration is the way which always works.
 I hoped to be able to it an it works in this package. (I checked it again without `usedevelop` and the paths did not get combined.)
But for zopefoundation/zope.securitypolicy#7 `usedevelop` did not help and I had to configure the paths in `.coveragerc`. I did not find any differences in the packages which seem to cause this different behaviour.

I'll leave it now as it is with this reminder for later.